recipes/ipyparallel/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "5.0.1" %}
+
+package:
+  name: ipyparallel
+  version: {{ version }}
+
+source:
+  fn: ipyparallel-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
```

Comment:
Straight to the source? https://github.com/ipython/ipyparallel/archive/5.0.1.tar.gz